rotkehlchen/globaldb/schema.py

```diff
@@ -230,6 +256,12 @@
 );
 """
 
+# asset_collections allows to set common information for related assets as descrbed in the
+# multiasset_mappings table. The reason to have custom name and symbol is that for some tokens
+# those properties are not the same across different chains. For example in gnosis chain USDC uses
+# USD//C as symnol for the USDC token. This table covers this problem create a common name and
+# symbol for those assets allowing a clean representation for the users and working as common
+# row to relate all the related asstes.
```

Comment:
Hm.... While reading I came up with a question. Shouldn't `coingecko`, `cryptocompare` and `protocol` be also shared among assets on multiple chains?

Reply:
I believe it makes sense for protocol. For the coingecko/cryptocompare I recall to have seen the asset in different chains with different oracle names. The reason to not have them in the assets_collections is because 1) you have to create a new collection for all the assets and in most of the cases will be a collection with one item 2) you add two extra queries to get those properties for every asset

Reply:
> For the coingecko/cryptocompare I recall to have seen the asset in different chains with different oracle names.

If this is true, alright.

In general, the idea that came to my mind is to remove all the data duplicates from the db. For example, if a token is in a multiasset collection we are going to always use name from the collection and not from `common_asset_details`, right? But this would probably be a too heavy change to do right now.